scalereg/simple_cfp/views.py

```diff
@@ -1,5 +1,5 @@
 from django.conf import settings
-from django.forms.util import ErrorList
+from django.forms.utils import ErrorList
```

Comment:
WDYT about just dropping the apps we are not using to reduce the maintenance burden?

Reply:
im not using these elsewhere either.  happy to drop them.

Reply:
rebased on your PR, so these are gone now.